workbench/app/Models/Tag.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Workbench\App\Models;
+
+use CalebDW\Laraflake\Concerns\HasSnowflakes;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Relations\MorphTo;
+
+class Tag extends Model
+{
+    use HasSnowflakes;
+
+    protected $guarded = [];
+
+    /**
+     * Get the taggable model that the tag belongs to.
+     *
+     * @return MorphTo<Model, $this>
+     */
```

Comment:
Prefer not to have obvious comments:


```suggestion
    /** @return MorphTo<Model, $this> */
```